cliquet/resource.py

```diff
@@ -255,9 +255,9 @@ class BaseResource(object):
     mapping = ResourceSchema()
     """Schema to validate records."""
 
-    def __init__(self, request, context):
+    def __init__(self, request, context=None):
```

Comment:
I see no test that makes sure resource constructor context is optional, due a bug in Cornice here https://github.com/mozilla-services/cornice/blob/master/cornice/service.py#L535-L536